### PR TITLE
Create Recipe Form; Input fields & Dropdown Select

### PR DESCRIPTION
**Pull request by mglavey**
### Ticket Number:
Create Recipe Form- this pull request is for the input fields for recipe title and description, respectively, in addition to the dropdown select menu for recipe duration, difficulty, and serving. 

### Checklist For Submitter

* [ ] I have documented all new functionality
* [ ] I have screenshotted new UI changes and attached them to this PR

### CODE REVIEW

File: frontend/src/components/EditRecipe/EditRecipeForm.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          <div className="form-title col-4 offset-4">
+            <label htmlFor="title">
+Recipe Title
+              <input className="form-control input-sm" id="title" type="text" onChange={this.handleInputChange} />
+            </label>
+          </div>
+          <div className="form-snippet col-6 offset-3">
+            <label htmlFor="snippet">
+Recipe Description
+              <textarea className="form-control" id="snippet" rows="4" onChange={this.handleInputChange} />
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.js]: I might make the recipe title and the description field about 50% wider, and maybe bump up the font size within the form elements themselves a bit. That'll still do what we talked about while still sticking a liiiiittle closer to the original design, at least.

File: frontend/src/components/EditRecipe/EditRecipeForm.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            </label>
+          </div>
+        </div>
+        <div className="row form-dropdown">
+          <div className="dropdown col-2 offset-3">
+            <h2>
+              <img src="https://img.icons8.com/material/26/000000/bar-chart.png" alt="difficulty" />
+Difficulty
+            </h2>
+            <select className="difficulty">
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.js]: These are still styled the default way. The ones on the design look more like the other input fields, with the grey rounded rectangle, some padding, a bigger font size, etc. If you were planning to come back to them, that's cool.

File: frontend/src/components/EditRecipe/EditRecipeForm.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          </div>
+          <div className="dropdown col-2">
+            <h2>
+              <i className="fas fa-utensil-spoon" />
+Servings
+            </h2>
+            <select className="servings">
+              <option>4</option>
+              <option>8</option>
+              <option>12</option>
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.js]: I might add, 2, 6, and 10 here.

File: frontend/src/components/EditRecipe/EditRecipeForm.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            <select className="duration">
+              <option>30 mins</option>
+              <option>60 mins</option>
+              <option>90 mins</option>
+              <option>120 mins</option>
+            </select>
+          </div>
+          <div className="dropdown col-2">
+            <h2>
+              <i className="fas fa-utensil-spoon" />
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.js]: Not sure how easy it'd be, but if we can adjust the line height (or whatever) of the spoon so it hits more toward the middle of the text, that'd be awesome.

Comment (mglavey) [reply frontend/src/components/EditRecipe/EditRecipeForm.js]: It's taking me quite a while to figure that out, I've tried a couple of different options to now avail, if anyone else wants to jump in on this.

**Comment by mglavey** [comment]
image

File: frontend/src/components/EditRecipe/EditRecipeForm.js (hunk 4 above)
Comment (H12) [reply frontend/src/components/EditRecipe/EditRecipeForm.js]: You could maybe drop this in your EditRecipeForm.css?
```css
.editRecipeForm i.fas.fa-utensil-spoon {
    padding-right: 15px;
}
```

Comment (mglavey) [reply frontend/src/components/EditRecipe/EditRecipeForm.js]: Ah ha! That's the solution, thanks @H12!

**Comment by mglavey** [comment]
image

**Pull request merged**